totp/__init__.py

```diff
@@ -20,6 +22,33 @@ def get_length(pass_entry):
     return 6
 
 
+def add_pass_entry(path):
+    """Add a new entry via pass."""
+    code_path = "2fa/{}/code"
+    code_path = code_path.format(path)
+
+    token_length = input('Token length [6]: ')
+    token_length = int(token_length) if token_length else 6
+
+    shared_key = getpass.getpass('Shared key: ')
+
+    pass_entry = "digits: {}\n{}".format(token_length, shared_key)
```

Comment:
I'm not sure what you mean here, because when I try to invert these lines, I get `UnboundLocalError: local variable 'shared_key' referenced before assignment` as I would expect. I also don't use `shared_key` anywhere else but these two lines.